packages/mdc-select/mdc-select.scss

```diff
@@ -44,55 +40,38 @@
 
   display: inline-flex;
   position: relative;
-  align-items: center;
-  justify-content: flex-start;
   box-sizing: border-box;
   height: 48px;
-  border: none;
-  outline: none;
   background-repeat: no-repeat;
-  background-position: right 8px bottom 8px;
-  cursor: pointer;
-  overflow: visible;
+  background-position: right 8px bottom 12px;
+  overflow: hidden;
 
   @include mdc-rtl {
-    background-position: left 8px bottom 8px;
-  }
-
-  &__menu {
-    position: fixed;
-    top: 0;
-    left: 0;
-    max-height: 100%;
-    transform-origin: center center;
-    z-index: 4; // Should pop up above everything else. temporary-drawer is next highest at 3.
+    background-position: left 8px bottom 12px;
   }
 
-  &__surface {
+  &__native-control {
     @include mdc-rtl-reflexive-property(padding, 0, $mdc-select-arrow-padding);
-    @include mdc-states;
+    @include mdc-typography-base;
+
+    &::-ms-expand {
+      display: none;
+    }
+
+    // counteracts the extra text padding that Firefox adds by default
+    @-moz-document url-prefix("") {
+      text-indent: -2px;
+    }
 
-    display: flex;
-    position: relative;
-    flex-grow: 1;
     width: 100%;
-    height: 48px;
+    padding-top: 22px;
     padding-bottom: 8px;
     border: none;
     outline: none;
-    overflow: hidden;
-  }
-
-  &__selected-text {
-    display: flex;
-    position: absolute;
-    bottom: 8px;
-    align-items: flex-end;
-    transition:
-      mdc-animation-exit-temporary(opacity, 125ms),
-      mdc-animation-exit-temporary(transform, 125ms);
+    background-color: transparent;
     white-space: nowrap;
-    overflow: hidden;
+    cursor: pointer;
+    appearance: none;
```

Comment:
FYI, across the code base the prefixed version of appearance is used

```
-moz-appearance: none;
-webkit-appearance: none;
```

According to https://caniuse.com/#search=appearance the unprefixed version has very little support currently

Reply:
The vendor prefixes are added automatically by autoprefixer when we build releases (and you are encouraged to do the same if you have your own Sass workflow). I have noticed that we have a few components using the vendor prefixes directly, which we should probably fix to use the non-prefixed version that will be picked up by autoprefixer.